### DISCUSSION TITLE
What is the state of the project?

### DISCUSSION THREAD

**Discussion by mberndt123** [ThoughtWorksInc/Binding.scala#424] [General]
Hi @Atry,

I think that Binding.scala is an extremely interesting project and a very promising foundation for the future of Scala GUI programming, whether in the Browser or in some other context.

That said, I'm confused about the state of the project. There is no README.md on the main branch that explains how this project is supposed to be used, and there also doesn't seem to be any documentation other than the Scaladoc. I've also noticed that the 12.x branch is more than 1000 commits ahead of main, and yet, recent development seems to be happening on the main branch.

Could you shed a bit of light on what's going on with the project? I'm afraid that the lack of documentation is something that will make potential users hesitant to consider Binding.scala for their projects, and that would be a shame.

**Comment by Atry** [comment]
The `main` branch is for 13.x, which is under development, not ready for public use at the moment. The 12.x branch is the latest stable version.

**Comment by mberndt123** [comment]
Hey @Atry, thanks for responding! Would you like to shed a bit more light on how 13.x is going to be different from 12.x and what the goals for this new release are?

**Reply by Atry** [reply]
13.x aims to support Scala 3 and mutual recursive data-binding. Also the implementation will be switched to `scalaz.StreamT` based. See https://twitter.com/pop_atry/status/1462009813139800064
I will publish an announcement when it's ready.

**Reply by mberndt123** [reply]
Cool, thanks for clearing that up!
